tracing/tracing.go

```diff
@@ -11,14 +11,14 @@ import (
 )
 
 // installJaeger registers Jaeger as the OpenTracing implementation.
-func installJaeger(serviceName string, cfg *jaegercfg.Configuration) io.Closer {
+func installJaeger(serviceName string, cfg *jaegercfg.Configuration) (io.Closer, error) {
 	metricsFactory := jaegerprom.New()
 	closer, err := cfg.InitGlobalTracer(serviceName, jaegercfg.Metrics(metricsFactory))
 	if err != nil {
 		fmt.Printf("Could not initialize jaeger tracer: %s\n", err.Error())
-		os.Exit(1)
+		return nil, err
```

Comment:
Return the error, wrapped, rather than logging it here.

Reply:
Sure